performance-storage-service/pss_project/api/serializers/rest/metrics/OLTPBenchMetricsSerializer.py

```diff
@@ -1,11 +1,12 @@
-from rest_framework.serializers import Serializer, DecimalField
+from rest_framework.serializers import Serializer, DecimalField, JSONField
 from pss_project.api.serializers.rest.metrics.LatencyMetricsSerializer import LatencyMetricsSerializer
 from pss_project.api.models.rest.metrics.OLTPBenchMetrics import OLTPBenchMetrics
 
 class OLTPBenchMetricsSerializer(Serializer):
     # Fields
     throughput = DecimalField(max_digits=24, decimal_places=15, coerce_to_string=False)
     latency = LatencyMetricsSerializer(required=False)
+    incremental_metrics = JSONField()
```

Comment:
I think it might be good to create an `IncrementalMetricsSerializer`. That way we can validate the shape of the JSON better.

Reply:
Done.